### PR TITLE
add module github.com/codegangsta/cli

### PR DESCRIPTION
**Pull request by pankaj1879**
## What type of PR is this?

_(REQUIRED)_

- bug
- cleanup
- documentation
- feature

## What this PR does / why we need it:

_(REQUIRED)_

## Which issue(s) this PR fixes:

_(REQUIRED)_

## Special notes for your reviewer:

_(fill-in or delete this section)_

## Testing

_(fill-in or delete this section)_

## Release Notes

_(REQUIRED)_

```release-note

```

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,5 +1,5 @@
 module github.com/urfave/cli/v3
-
+module github.com/codegangsta/cli
```
Comment (dearchap) [inline go.mod:2]: @pankaj1879 Why do we need this ? the codegangsta/cli corresponds to v1 of this module.

Comment (pankaj1879) [reply go.mod:2]: facing this error 
github.com/codegangsta/cli: github.com/codegangsta/cli@v1.22.10: parsing go.mod:
	module declares its path as: github.com/urfave/cli
	        but was required as: github.com/codegangsta/cli


please help me

Comment (dearchap) [reply go.mod:2]: @pankaj1879 You need to update your go.mod as follows

```
replace github.com/codegangsta/cli v1.22.10 => github.com/urfave/cli
```

**Comment by dearchap** [comment]
@pankaj1879 Please look at golang module guidelines for how to proceed. 

https://go.dev/ref/mod#:~:text=A%20replace%20directive%20replaces%20the,a%20platform%2Dspecific%20file%20path.

**Pull request closed without merging**